source/IAccessibleHandler.py

```diff
@@ -1014,6 +1021,11 @@ def splitIA2Attribs(attribsString):
 	@return: A dict of the attribute keys and values, where values are strings or dicts.
 	@rtype: {str: str or {str: str}}
 	"""
+	# Do not treat huge base64 data as it might freeze Google Chrome (#10227)

```

Comment:
This comment should say Freeze NVDA not freeze Google Chrome.

Reply:
Fixed as of d778b7946
Same fix reported on the [Python 2 backport branch](/accessolutions/nvda/tree/i10227-ia2Base64-py2) as of commit e0e35e2a66